src/com/twu/biblioteca/MainMenuImplementation.java

```diff
@@ -68,7 +74,21 @@ public void displayAvailableBooks() {
             }
         }
     }
-
+/////////////////////////////
+    public void displayAvailableMovies() {
+      //  if (this.movies.size() == countCheckedOutMovies()) {
+      //      showNoMoviessMessage();
+      //  } else {
+            for (int i = 0; i < this.movies.size(); i++) {
+                Movie movie = this.movies.get(i);
+               // if (!movie.isCheckedOut()) {
+                    int reference = i + 1;
+                    System.out.println(reference + ". " + movie);
+                //}
+            //}
+        }
+    }
+///////////////////////////////////
```

Comment:
Nice work on all this!  Let's talk about what is common between `Book` and `Movie` tonight, and how we can use that commonality to _not_ need separate methods to display each of the lists.